scripts/peering-bgp

```diff
@@ -69,7 +69,10 @@ cli)
     $birdc_exec -s "$bird_sock"
     ;;
 start)
-    test -S "$bird_sock" && die "error [BIRD control socket exists]"
+    if [ -S "$bird_sock" ]; then
```

Comment:
Please submit this change in a separate PR in a different branch so we can merge it immediately. (This PR is on hold until we merge and deploy the API to the website.)